doc/samtools-mpileup.1

```diff
@@ -129,7 +129,7 @@ Forward	Reverse	Meaning
 \&.\fR dot	,\fR comma	Base matches the reference base
 ACGTN	acgtn	Base is a mismatch to the reference base
 >	<	Reference skip (due to CIGAR \(lqN\(rq)
-*	*	Deletion of the reference base (CIGAR \(lqD\(rq)
+*	*(#)	Deletion of the reference base (CIGAR \(lqD\(rq)
```

Comment:
Thanks. A couple of comments on the man page:

Where options affect the pileup output, I put a pointer to them in the pileup description — something like (don't just apply this as the `.TE` will get duplicated, but you get the idea :smile:)

```suggestion
*      *\fR/\fB#       Deletion of the reference base (CIGAR \(lqD\(rq)
 .TE

Deleted bases are shown as \(lq*\(rq on both strands
unless \fB--reverse-del\fR is used.

```